servers/publikator/lib/cache/search.js

```diff
@@ -13,7 +13,7 @@ const client = elasticsearch.client()
 const getSort = (args) => {
   // Default sorting
   if (!args.orderBy) {
-    return { sort: { 'latestCommit.date': 'desc' } }
+    return
```

Comment:
`args.search ? undefined : { sort: { 'latestCommit.date': 'desc' } }`, IMO you still want `'latestCommit.date': 'desc'` when using it without search (list on publikator index page)